### ISSUE TITLE
Center Hot/Cold clue locations

### ISSUE THREAD

**Issue by TheStonedTurtle** [runelite/runelite#5073]
Hot/Cold steps have a 9x9 location where it registers the dig as correct. Some of our locations are not centered correctly on this location.

By centering all of these locations it may fix that bug where sometimes it will eliminate all end locations accidentally.

<details><summary>Image Example</summary>
<p>

![](https://i.imgur.com/bLj1XS6.png)

</p>
</details>

<hr/>

## How to determine Center Location

* 1) Use the Strange Device to find one of the four corners.
* 2) Once you have figured out one of the 4 corners determine the other 4.
  * Use `Ground Markers` and Shift+Right-click mark the corner tiles to help keep track of corners.
* 3) Once you have figured out all 4 corners walk 4 squares diagonally inside the square.
* 4) Now that you are on the center square get the Location by using the Developer Tools.
  * The `Tile` line is what we care about: ![](https://i.imgur.com/hvnXAW6.png)
* 5) Share this information on this Issue using the below format.

## Format
| Area Text | New Tile Info
|- |-
| In the cow pen north of Sarah's Farming Shop | 3031, 3305, 0

Copy-Paste format
```
| Area Text  | New Tile Info
|- |-
| In the cow pen north of Sarah's Farming Shop | 3031, 3305, 0
```

<hr/>

## Updated Locations

| Area Text | New Tile Info
|- |-
| In the cow pen north of Sarah's Farming Shop | 3031, 3305, 0

**Comment by xDemoN** [comment]
This can be closed in favor of #9601

**Comment by Nightfirecat** [comment]
Agreed, 9601 has a better list for tracking fixes to these.
